cross/lame/Makefile

```diff
@@ -13,6 +13,29 @@ COMMENT  = High quality MPEG Audio Layer III encoder
 LICENSE  = LGPL
 
 GNU_CONFIGURE = 1
+CONFIGURE_ARGS += --with-Dll=no --with-debian=no
```

Comment:
These options don't exist in this version.
That why I remove `debian` in the patch : https://github.com/SynoCommunity/spksrc/pull/1662/files#diff-f49f93733bb8adef53d3f4dfe392792fR188